docs/resources/slint-docs-preview.html

```diff
@@ -39,6 +39,11 @@
     }
 
     function should_show_automatic_preview(element) {
+        // Safari blocks quite heavily when creating many WebGL contexts, slowing down loading of the docs
+        // significantly. For now, don't do automatic previews:
+        if (navigator.userAgent.indexOf('Safari') != -1 && navigator.userAgent.indexOf('Chrome') == -1) {
```

Comment:
Why is there the `indexOf('Chrome')` .  Looks like an error to me.

Reply:
Because Chrome on macOS includes both strings (Safari and Chrome) in its user agent. See also the table at https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent#which_part_of_the_user_agent_contains_the_information_you_are_looking_for

Reply:
So that only blocks Chrome on macOS,  but the commit says it blocks Safari.